generator/Makefile

```diff
@@ -118,6 +119,14 @@ $(MIBDIR)/IANA-CHARSET-MIB.txt:
 	@echo ">> Donwloading IANA charset MIB"
 	@curl -s -o $(MIBDIR)/IANA-CHARSET-MIB.txt -L $(IANA_CHARSET_URL)
 
+$(MIBDIR)/IANA-IFTYPE-MIB.txt:
+# We donwload the ifType MIB from IANA instead of using the one that comes
```

Comment:
IMO this comment is unnecessary, as this is the official source. We only used the one in the Cisco tree out of laziness/convenience.

Reply:
Ya, I think that makes more sense. I'll wait until you and @brian-brazil reach consensus and update accordingly.

Reply:
Go with SuperQ here.